MovieTime/MovieTime/Model/Services/MTResponse.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  MTResponse.swift
+//  MovieTime
+//
+//  Created by Enrique Melgarejo on 08/01/18.
+//  Copyright © 2018 Choynowski. All rights reserved.
+//
+
+import Foundation
+
+final class MTResponse: CustomDebugStringConvertible, Equatable {
+
+    let statusCode: Int
+    let data: Data?
+    var error: MTNetworkError?
```

Comment:
why is this property mutable while the others are not?

Reply:
Because I have not replace the `var` to `let` when I change the code.
Initially, the http status code validation was like the method bellow, but I switched to the current implementation because it is simpler, readable, covers more different types of errors and is testable:
```
func filterStatusCodes(_ range: ClosedRange<Int>) -> MTResponse {
	guard range.contains(statusCode) else {
		self.error = MTNetworkError.unexpectedError(message: "A service error happened.")
		return self
	}
	return self
}
```